src/GraphQL/Types/Input/BeamPackInputType.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Enjin\Platform\Beam\GraphQL\Types\Input;
+
+use Rebing\GraphQL\Support\Facades\GraphQL;
+
+class BeamPackInputType extends InputType
+{
+    /**
+     * Get the input type's attributes.
+     */
+    public function attributes(): array
+    {
+        return [
+            'name' => 'BeamPackInput',
+            'description' => __('enjin-platform-beam::input_type.beam_pack.description'),
+        ];
+    }
+
+    /**
+     * Get the input type's fields.
+     */
+    public function fields(): array
+    {
+        return [
+            'id' => [
+                'type' => GraphQL::type('Int'),
+                'description' => __('enjin-platform-beam::input_type.beam_pack.field.id'),
+            ],
+            'tokens' => [
+                'type' => GraphQL::type('[ClaimToken!]!'),
+                'description' => __('enjin-platform-beam::input_type.claim_token.description'),
+            ],
+        ];
```

Comment:
what will be the `claimQuantity` limit?

Reply:
Also this field should only be prohibited if all the tokens are MINT_ON_DEMAND right?

Reply:
I think it can apply to either mint on demand or transfer, the distribution method probably doesn't matter, it's just a shortcut to reduce the amount of repeated input data for multiples of the same FT pack.  The `claimQuantity` limit would probably be the collection max token count, taking onto consideration the already minted tokens allocated to beams etc., similar to the existing `MaxTokenCount` rule.

Reply:
I think we'll have problem if it's TRANSFER_TOKEN, once transferred the wallet daemon no longer have it. So I think its not possible to repeat the token when its TRANSFER_TOKEN.

Reply:
For FT that's fine, as long as the wallet daemon has more balance of the tokens in the pack then it can keep transferring, all this does is help to reduce the amount of repeated data required for creating the packs, so rather than this:
```packs: [
      {
        tokens: {
          tokenIds: ["1..3"]
          tokenQuantityPerClaim: 1
          type:TRANSFER_TOKEN
        }
      },
      {
        tokens: {
          tokenIds: ["1..3"]
          tokenQuantityPerClaim: 1
          type:TRANSFER_TOKEN
        }
      },
      {
        tokens: {
          tokenIds: ["1..3"]
          tokenQuantityPerClaim: 1
          type:TRANSFER_TOKEN
        }
      }
    ]
``` 
We could do this:
```
packs: [
      {
        claimQuantity: 3,
        tokens: {
          tokenIds: ["1..3"]
          tokenQuantityPerClaim: 1
          type:TRANSFER_TOKEN
        }
      }
    ]
```
In both we are saying that there are 3 packs that can be claimed, each pack transfers a balance of 1 each of token id 1, 2 and 3. Internally all we'd do is expand the second version `claimQuantity` number of times, so we create the same data in the DB either way, it's just a bit less input data in the second one.